src/components/Explore/ByFrameRepeater.tsx

```diff
@@ -72,7 +72,7 @@ export class ByFrameRepeater extends SceneObjectBase<ByFrameRepeaterState> {
                 new SceneCSSGridLayout({
                   children: [
                     new ErrorStateScene({
-                      message: data.data.errors?.[0]?.message ?? 'An error occurred in the query',
+                      message: (data.data.errors?.[0]?.message || data.data.error?.message) ?? 'An error occurred in the query',
```

Comment:
[This field is deprecated](https://github.com/grafana/grafana/blob/v11.6.0/packages/grafana-data/src/types/panel.ts#L65), we should probably not rely on it.